docs/reference/search/search-your-data/retrievers_examples.asciidoc

```diff
@@ -0,0 +1,327 @@
+tag::basic-rrf-retriever-with-semantic-query[]
+[discrete]
+=== Combining kNN and semantic search with RRF
```

Comment:
Shouldn't it be more interesting to combine BM25 with semantic / kNN for hybrid search?

Reply:
Good point - IMO having something in multiple places is OK as long as it is relevant on the current topic and can be accessed through different use cases (RRF is specific, retrievers is more of a framework). 

For me it doesn't make a lot of sense to combine semantic query with knn - they are basically different ways of achieving vector search, I'm not sure if I would use it in the Real World ™️ . But hybrid is something I definitely would.

Reply:
I see what you mean. Updated to have a `standard` retriever with a `query_string` query and a `knn` retriever.

Reply:
Will also remove the initial ELSER setup now that we don't directly reference the `semantic` query anymore.